packages/css/src/components/Modal/modal.css

```diff
@@ -0,0 +1,52 @@
+.modal {
+    position: absolute;
+    max-width: 660px;
+    width: 100%;
+    border-radius: var(--modal-border-radius);
+    background: var(--modal-background-color);
+    box-shadow: var(--modal-shadow);
+}
+
+.modal-header {
+    padding: var(--padding-3) var(--padding-5);
+    border-bottom: 1px solid var(--modal-border-color);
+    color: var(--modal-header-color);
+    font: var(--modal-header-font);
+}
+
+.modal-body {
+    border-bottom: 1px solid var(--modal-border-color);
+    color: var(--modal-body-color);
+    font: var(--modal-body-font);
+}
+
+.modal-footer {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    align-items: center;
+    padding: var(--padding-3) var(--padding-5);
+    gap: var(--margin-3);
+}
+
+.modal-button-close {
+    position: absolute;
+    top: calc(var(--padding-3) + 4px);
+    right: var(--padding-5);
+    font-size: 24px;
+    cursor: pointer;
+}
+
+.modal-button-close svg {
+    display: block;
+    width: 1em;
+    height: 1em;
+}
+
+.modal.small {
+    max-width: 320px;
```

Comment:
Same as other size comments - we shouldn't be hard coding values
If we have to hardcode something we should at least keep track of such items 

Can we make a file for manually created css vars and move these there 

This way we have a vars file that lists all the variables we had to come up with, they can be moved to the design system at some point